controller/work_item_link_blackbox_test.go

```diff
@@ -103,8 +103,10 @@ func (s *workItemLinkSuite) cleanup() {
 	require.Nil(s.T(), db.Error)
 	db = db.Unscoped().Delete(&link.WorkItemLinkCategory{Name: "test-user"})
 	require.Nil(s.T(), db.Error)
-	db = db.Unscoped().Delete(&space.Space{ID: s.userSpaceID})
-	require.Nil(s.T(), db.Error)
+	if s.userSpaceID != uuid.Nil {
```

Comment:
@sbose78 I don't understand the title of this PR. Should this line just remove the `s.userSpaceID` which is a new created space. It isn't related to `system.space` afaik, https://github.com/sbose78/almighty-core/blob/511b06cd32a9413ac60f837dcdf1e5c6e30c9f5c/controller/work_item_link_blackbox_test.go#L165. 
Am I missing something :) ?